### DISCUSSION TITLE
Add quicker way to add multiple users to a given role

### DISCUSSION THREAD

**Discussion by erondpowell** [directus/directus#15343] [Feature Requests]
### Describe the Bug

<img width="731" alt="Screen Shot 2022-08-31 at 11 46 22 AM" src="https://user-images.githubusercontent.com/62306630/187735357-3ef69205-ceb0-4d65-bba9-322a5f97954c.png">

When you assign a role to a user, you ahve to double-confirm to save it.
A small thing but seems like a UX improvement.

### To Reproduce

UI issue

### Errors Shown

UI issue

### What version of Directus are you using?

n/a

### What version of Node.js are you using?

n/a

### What database are you using?

n/a

### What browser are you using?

n/a

### How are you deploying Directus?

n/a

**Comment by rijkvanzanten** [comment]
@erondpowell Is that the correct screenshot? 🤔

**Comment by rijkvanzanten** [comment]
Ohhh you mean that the o2m opens a drawer of items?

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
